### PR TITLE
idk some minor spelling mistakes fixed engine ruined?

### PR DESCRIPTION
**Pull request by Jotaro-Gaming**
idk

### CODE REVIEW

**Comment by DaBanana64** [comment]
Psych Engine users when they see minor spelling mistakes: OMG ENGINE RUINED!

Kade Engine users when their fucking game randomly fucking crashes: WOW GOOD ENGINE!!!!!

**Comment by trayfellow** [comment]
image

**Comment by HiroMizuki** [comment]
i think the bad spelling doesn't really ruins the engine, spelling is just something that gives a good look, it doesn't change the functionality or smth else

**Comment by MrClogsworthYT** [comment]
> i think the bad spelling doesn't really ruins the engine, spelling is just something that gives a good look, it doesn't change the functionality or smth else

ye ye

File: README.md [hunk 1]
```diff
@@ -41,17 +41,17 @@ _____________________________________
...
+  * BF does a "Hey!" during Philly Nice
   * Blammed has a cool new colors flash during that sick part of the song
 ### Week 4:
-  * Better hair physics for Mom/Boyfriend (Maybe even slightly better than Week 7's :eyes:)
-  * Henchmen die during all songs. Yeah :(
+  * Better hair physics for Mom and Boyfriend (Maybe even slightly better than Week 7's :eyes:)
+  * Henchmen die during all the songs. Yeah :(
 ### Week 5:
-  * Bottom Boppers and GF does "Hey!" animations during Cocoa and Eggnog
+  * Bottom Boppers and GF does a "Hey!" animation during Cocoa and Eggnog
```
Comment (lemz1) [inline README.md:54]: change does to do.
Line 54.

Comment (MrClogsworthYT) [reply README.md:54]: > change does to do.
> 
> Engine ruined

it wont make sense bruh
it will be `gf do a "Hey!"` animations

Comment (lemz1) [reply README.md:54]: It is bottom boppers and gf.
It is plural

Comment (MrClogsworthYT) [reply README.md:54]: > It is bottom boppers and gf. It is plural

fine

**Comment by tposejank** [comment]
shadow mario is brazilian guys, dont expect him to be god at spelling

**Comment by HiroMizuki** [comment]
> shadow mario is brazilian guys, dont expect him to be god at spelling

true

**Comment by DaBanana64** [comment]
Just because this Engine has some minor spelling mistakes doesn't mean it ruins the whole Engine lmfao.

File: README.md [hunk 2]
```diff
@@ -41,17 +41,17 @@ _____________________________________
   * New Dad Left sing sprite 
   * Unused stage lights are now used
 ### Week 2:
-  * Both BF and Skid & Pump does "Hey!" animations
-  * Thunders does a quick light flash and zooms the camera in slightly
+  * Both BF Skid & Pump do a "Hey!" animation
```
Comment (Steviegt6) [inline README.md:44]: "Both BF and Skid & Pump" is more grammatically correct than this. Changing "does" to "do a" is correct, but the removal of "and" forms a list without commas. Previously, "BF" and "Skid & Pump" were treated as separate items in a list, now that distinction is unclear.

File: README.md [hunk 3]
```diff
@@ -41,17 +41,17 @@ _____________________________________
...
+  * Thunders do a quick light flash and zooms the camera in slightly
   * Added a quick transition/cutscene to Monster
 ### Week 3:
-  * BF does "Hey!" during Philly Nice
+  * BF does a "Hey!" during Philly Nice
   * Blammed has a cool new colors flash during that sick part of the song
 ### Week 4:
-  * Better hair physics for Mom/Boyfriend (Maybe even slightly better than Week 7's :eyes:)
-  * Henchmen die during all songs. Yeah :(
+  * Better hair physics for Mom and Boyfriend (Maybe even slightly better than Week 7's :eyes:)
```
Comment (Steviegt6) [inline README.md:51]: This wasn't a grammar issue, really -- personal preference?

File: README.md [hunk 4]
```diff
@@ -41,17 +41,17 @@ _____________________________________
...
+  * Henchmen die during all the songs. Yeah :(
```
Comment (Steviegt6) [inline README.md:52]: Frankly unnecessary article inclusion.

File: README.md (hunk 1 above)
Comment (Steviegt6) [inline README.md]: Could easily get away with changing "does" to "do". "a" implies that there is a singular animation played, while it's actually played throughout the song.

(This applies to practically every other change you made regarding the "Hey!" animations.)

**Review by steviegt6 — CHANGES_REQUESTED** [review]
Negligible grammatical errors detected, pull request ruined.

File: README.md (hunk 2 above)
Comment (MrClogsworthYT) [reply README.md]: > "Both BF and Skid & Pump" is more grammatically correct than this. Changing "does" to "do a" is correct, but the removal of "and" forms what appears to be a list without commas. Previously, "BF" and "Skid & Pump" were treated as separate items, now that distinction is unclear.

resolved

File: README.md (hunk 4 above)
Comment (MrClogsworthYT) [reply README.md:52]: > Frankly unnecessary article inclusion.

ok 👍

File: README.md (hunk 3 above)
Comment (MrClogsworthYT) [reply README.md]: resolved

**Comment by ShadowMario** [comment]
Dude, this is a Readme file, nobody cares.

**Comment by MrClogsworthYT** [comment]
> Dude, this is a Readme file, nobody cares.

k

**Pull request closed without merging**